test/e2e/framework/framework.go

```diff
@@ -105,26 +110,67 @@ func newFramework(kubeconfig string, qps float32, burst int) (*Framework, error)
 	}, nil
 }
 
+const (
+	kubeconfigFlag      = "kubeconfig"
+	gsimageFlag         = "gameserver-image"
+	pullSecretFlag      = "pullsecret"
+	stressTestLevelFlag = "stress"
+	perfOutputDirFlag   = "perf-output"
+	versionFlag         = "version"
+)
+
+// ParseTestFlags Parses go test flags separately because pflag package ignores flags with '-test.' prefix
+// Related issues:
+// https://github.com/spf13/pflag/issues/63
+// https://github.com/spf13/pflag/issues/238
+func ParseTestFlags() error {
```

Comment:
It is a bit hacky way, but since this is for test and we have not send a PR to spf13 repo this is an acceptable and small workaround.